ci/tag-maintainers/manage-reviewers.py

```diff
@@ -0,0 +1,234 @@
+#!/usr/bin/env python3
+"""
+Manage pull request reviewers for Nixvim.
+
+This script handles the reviewer management logic from the tag-maintainers workflow,
+including checking for manually requested reviewers and managing removals.
+"""
+
+import argparse
+import json
+import logging
+import subprocess
+import sys
+from typing import Final
+
+# Configure logging to output to stderr
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(message)s",
+    stream=sys.stderr,
+)
+
+MANUAL_REVIEW_REQUEST_QUERY: Final[str] = """
+query($owner: String!, $repo: String!, $prNumber: Int!) {
+  repository(owner: $owner, name: $repo) {
+    pullRequest(number: $prNumber) {
+      timelineItems(first: 100, itemTypes: [REVIEW_REQUESTED_EVENT]) {
```

Comment:
I wonder if it's more relevant to have the first or last 100 timeline items?

Reply:
Yeah, I am also curious if there is any harm to setting this to a higher number where it doesn't matter.

Reply:
Bumped to 250 (max allowed by their graphql endpoint) and switched to last.

Reply:
> But, it would be nice to eventually refactor this to be smarter about looking at the current state of the reviewers and query for their specific review history maybe? 

While that may be more correct, I think the current implementation better optimises for making less API requests. I think that's more important, given how unlikely it is we'll have over 250 review request events.